litestar/dto/factory/types.py

```diff
@@ -70,3 +70,6 @@ def make_field_type(self, inner_type: type) -> Any:
 
 FieldDefinitionsType: TypeAlias = "Mapping[str, FieldDefinition | NestedFieldDefinition]"
 """Generic representation of names and types."""
+
+RenameStrategy: TypeAlias = Union[Literal["lower", "upper", "camel", "pascal"], Callable[[str], str]]
```

Comment:
```suggestion
RenameStrategy: TypeAlias = 'Literal["lower", "upper", "camel", "pascal"] | Callable[[str], str]'
```
Using forward references means we can use the nicer union syntax.

Reply:
Neat, thanks. Didn't know that would work in py3.8